app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+
+from .errors import NotVaccinatedError
+from .errors import OutdatedVaccineError
+from .errors import NotWearingMaskError
```

Comment:
The checklist recommends using absolute imports instead of relative imports. Consider changing these imports to absolute imports, e.g., from app.errors import NotVaccinatedError.